client/landing/stepper/declarative-flow/internals/steps-repository/components/setup-form/style.scss

```diff
@@ -91,13 +91,20 @@
 		// adds +2 px of padding to the button to match the input field height
 		padding: 10px 14px;
 		margin-top: 16px;
+		&.disabled,
+		&.disabled:focus,
+		&.disabled:hover,
+		&:disabled {
```

Comment:
I might be mixing repositories up but remember seeing `is-disabled` instead — maybe that would carry over styling  over from component itself, even if the button isn't _actually_ disabled?

Either way this is fine for merging.